drivers/sensor/vl53l1x/vl53l1_platform.c

```diff
@@ -0,0 +1,374 @@
+
+/*
+ * Copyright (c) 2017, STMicroelectronics - All Rights Reserved
+ * Copyright (c) 2019 Makaio GmbH
+ *
+ * This file is part of VL53L1 Core and is dual licensed,
```

Comment:
In zephyr main repo, except in ext/, you are only allowed to use Apache 2.0 Licence.